src/index.js

```diff
@@ -2,78 +2,95 @@ const Discord = require('discord.js')
 const { main, exec, js, shard, jsi } = require('./commands')
 const { codeBlock } = require('./utils')
 
-module.exports = class Dokdo {
-  /**
-     * @typedef {Function} noPerm
-     * @param {Discord.Message} message
-     * @returns {any|Promise<any>}
-     */
-  /**
-     * @typedef {Object} options
-     * @property {?string[]} aliases Aliases of command
-     * @property {?string[]} owners ID of owners
-     * @property {?string} prefix Prefix of Bot
-     * @property {?any[]} secrets Secrets to hide
-     * @property {?noPerm} noPerm Executed when command runned by not allowed user
-     */
+/**
+ * @typedef {(message: Discord.Message)} noPerm
+ * @returns {any|Promise<any>}
+*/
+
+/**
+ * @typedef options
```

Comment:
```suggestion
 * @typedef DokdoOptions
```

Because of index.d.ta typing